lang/python/setup_pip.py

```diff
@@ -35,8 +35,8 @@
 from __future__ import print_function
 import os, os.path, re, shutil, sys
 from setuptools import setup, Distribution, Extension
-import distutils.sysconfig
-import distutils.ccompiler
+# import distutils.sysconfig
+# import distutils.ccompiler
```

Comment:
Remove the lines, don't just comment.

Reply:
removed in most recent commit: [924e8e7](https://github.com/wiredtiger/wiredtiger/pull/10492/commits/924e8e795d33874fa216d570580193aa91958644)